README.md

```diff
@@ -1,22 +1,111 @@
-# Config file reader
+# Config parser
 
-This module reads project configuration for a snapshot run from a JSON file
-requested via http. The file schema is specified in `warhol.schema.json`. An
-example:
+This module validates project configuration objects or JSON strings and turns
+valid partial configuration objects into full configurations with sensible
+defaults. The file schema is specified in `warhol.schema.json`. An example:
 
 ```json
 {
-  "styleguideUrl": "https://foobar.com/",
+  "styleguideUrl": "https://foo.com/components",
   "breakpoints": [
     1280, 1024, 800, 400, 280
   ],
+  "theme": {
+    "colors": {
+      "colorsUrl": "https://foo.com/components/colors",
+      "sources": ".swatch",
```

Comment:
Is it allowed to use an array here or would you use a selector list like `.foo, .bar`?

Reply:
Only strings allowed here. Selector lists will work just fine *if* anybody ever needs more than just `.swatch` or similar.